tests/stress_tests/scripts/compare_memcheck_2_runs.py

```diff
@@ -47,26 +59,15 @@ def get_memcheck_records(source, db_collection=None, db_name=None, db_url=None):
     return items
 
 
-def prepare_comparison_table_csv(data, data_metrics, output_file):
-    """generate .csv file with table based on provided data"""
-    fields = list(data[0].keys())
-    metrics_names = list(data_metrics[0].keys())
-    HEADERS = fields + metrics_names
-    with open(output_file, 'w', newline="") as csvfile:
-        csvwriter = csv.writer(csvfile)
-        csvwriter.writerow(HEADERS)
-        for i in range(len(data)):
-            row = []
-            for field in fields:
-                row.append(data[i][field])
-            for metric_name in metrics_names:
-                row.append(data_metrics[i][metric_name])
-            csvwriter.writerow(row)
-
-
 def compare_memcheck_2_runs(cur_values, references, output_file=None):
     """Compares 2 MemCheckTests runs and prepares a report on specified path"""
 
+    returncode = 0
+
+    # constants
+    metric_name_template = "{} {}"
+    GEOMEAN_THRESHOLD = 0.9
```

Comment:
Isn't UPPER_CASE only for global variables? Please ensure pylint is passing.

Reply:
No, pylint also recognizes constants (and requires constants to be in UPPER_CASE), but I think it works strange because I mean `GEOMEAN_THRESHOLD` as constant (but pylint warns about it as local variable), and `exit_code` in bottom of file as local variable (but pylint requires to rename it according to constant UPPER_CASE).

So I ignore it